projects/aca-content/src/lib/components/bulk-actions-dropdown/bulk-actions-dropdown.component.ts

```diff
@@ -80,10 +83,20 @@ export class BulkActionsDropdownComponent implements OnInit, OnDestroy {
         this.tooltip = title;
         this.placeholder = placeholder;
       });
+
+    this.extensions.resetBulkActionsSubject$.pipe(takeUntil(this.onDestroy$)).subscribe(() => {
```

Comment:
Instead of injecting MatSelect using ViewChild, you can also use the disableControl form control, and set the value through that. Also, while you are at it, maybe rename the property to be something other than disableControl? Maybe something like bulkSelectControl

Reply:
Done, thanks!